substrate/frame/broker/src/lib.rs

```diff
@@ -716,55 +716,51 @@ pub mod pallet {
 
 		/// Drop an expired Region from the chain.
 		///
-		/// - `origin`: Must be a Signed origin.
+		/// - `origin`: Can be any kind of origin.
 		/// - `region_id`: The Region which has expired.
 		#[pallet::call_index(14)]
 		pub fn drop_region(
-			origin: OriginFor<T>,
+			_origin: OriginFor<T>,
 			region_id: RegionId,
 		) -> DispatchResultWithPostInfo {
-			let _ = ensure_signed(origin)?;
```

Comment:
But now there is no fee paid in the error case. Formerly it would "slash" the fee for wrong input params, now its free.

Reply:
Hmm? This doesn't change anything in regards to fee payment?

Reply:
Oh yea, its not "unsigned" 🤦‍♂️